src/Directive/ConfigurationBlockDirective.php

```diff
@@ -17,6 +17,27 @@
 
 class ConfigurationBlockDirective extends SubDirective
 {
+    private const LANGUAGE_LABELS = [
+        'env' => 'Bash',
```

Comment:
I would rather do sth. like:

```suggestion
    private const LANGUAGES = [
        [
            'type' => 'env',
            'label' => 'Bash',
        ],
```

In this case it will be much easier to extend it later if we need some special settings

Reply:
If this was a public package, I'd agree ... but since this is internal and only for Symfony Docs, we can change anything at any time (BC breaks don't apply to us in this project). So, I'd keep it like this.

Also, in other classes we have similar, but different, config -> https://github.com/weaverryan/docs-builder/blob/4080f308aa6359da1e9643ea18faf9d2c52daf9a/src/Renderers/CodeNodeRenderer.php#L24 so we should refactor everything to have a common config ... but Id' keep it like it is for now.